src/wallet/wallet.ts

```diff
@@ -147,6 +160,25 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     await handleCreate(data.status);
   }
 
+  async onNewTx(newTx: any) {
+    const outputs: any[] = newTx.outputs;
```

Comment:
Why type `any` here?

Reply:
Done in 1e3e15236ba33f9bc5fd3d8ca94cdb32c12ccb56